README.md

```diff
@@ -366,6 +366,71 @@ Check if an object is a Gotu Entity class
         entity.isEntity(Object) // false
 ```
 
+## Entity schema - `entity.schema`
+
+It is possible to access the schema information of your entity. There are two special properties:
+
+### fields
+
+Returns an list with all the fields of the entity:
```

Comment:
Would be good to add how isID works here and change the documentation in that section.

![image](https://user-images.githubusercontent.com/6617759/152343617-2301363f-a09c-4184-8b5a-a8529d1262ea.png)

Reply:
Done!